app-office/ledger/ledger-3.2.0.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit check-reqs cmake-utils python-single-r1 python-utils-r1 bash-completion-r1
+
+DESCRIPTION="A double-entry accounting system with a command-line reporting interface"
+HOMEPAGE="https://www.ledger-cli.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+IUSE="debug doc emacs python"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+RESTRICT="test"
+
+CHECKREQS_MEMORY=8G
+
+RDEPEND="
+	dev-libs/boost:=[python?]
+	dev-libs/gmp:0=
+	dev-libs/mpfr:0=
+	python? (
+		$(python_gen_cond_dep '
+			dev-libs/boost:=[${PYTHON_USEDEP}]
+			dev-python/cheetah3:=[${PYTHON_USEDEP}]
+		')
+		${PYTHON_DEPS}
+	)
+"
+DEPEND="
+	${RDEPEND}
+	dev-libs/utfcpp
+	doc? (
+		sys-apps/texinfo
+		virtual/texi2dvi
+		dev-texlive/texlive-fontsrecommended
+	)
+"
+
+# Building with python integration seems to fail without 8G available
+# RAM(!)  Since the memory check in check-reqs doesn't count swap, it
```

Comment:
This is no longer true:
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=297e41122a17dea6e7867ed4345255916c379a6c
This whole comment could be dropped.
`pkg_pretend()` could have implementation similar to `pkg_setup()`:
```
pkg_pretend() {
	if use python; then
		check-reqs_pkg_pretend
	fi
}
```

Reply:
Done.